eos/parameters/form-factors.yaml

```diff
@@ -673,6 +673,12 @@
     min:     -0.105
     max:     +0.165
     comments: ""
+# a value for B->K^*::alpha^A12_0 was missing previously, set it to zero default value for the moment
```

Comment:
@akokulu7 You will find that the form factor class does not use alpha^A12_0 at all. The reason is that the A12 form factor at q2 = 0 can be expressed through the A0 form factor; similarly also for T23. See [here](https://github.com/eos/eos/blob/v0.1.1/eos/form-factors/mesonic-impl.hh#L613) for the initialization of the parameters, and [here](https://github.com/eos/eos/blob/v0.1.1/eos/form-factors/mesonic-impl.hh#L660-L668) for the use in the A12 form factor.

Please remove this piece.